spec

```diff
@@ -4862,7 +4862,6 @@ beginseg
     name "ovl_Demo_Moonend"
     compress
     include "build/src/overlays/actors/ovl_Demo_Moonend/z_demo_moonend.o"
-    include "build/data/ovl_Demo_Moonend/ovl_Demo_Moonend.data.o"
     include "build/data/ovl_Demo_Moonend/ovl_Demo_Moonend.reloc.o"
```

Comment:
Jenkins is angry because you forgot to update the reloc, it should look like this
```suggestion
    include "build/src/overlays/actors/ovl_Demo_Moonend/ovl_En_Minislime_reloc.o"
```